### ISSUE TITLE
[Offline basemap download] Do not let user open "Select and download" when offline

### ISSUE THREAD

**Issue by gino-m** [google/ground-android#662]
Instead, show a modal dialog "Offline basemaps may only be downloaded when online" or similar.

**Comment by anandwana001** [comment]
@gino-m UX is needed for this task, right?

**Comment by gino-m** [comment]
> [@gino-m[<img alt="" width="15" height="15" src="chrome-extension://dlebflppeeemcdpidccbiblndppbmjmh/ospo-chrome-ext-logo.png">](https://teams.googleplex.com/gmiceli@google.com)](https://github.com/gino-m) UX is needed for this task, right?

I don't think so; we can show a toast using `EphemeralPopups` for both cases.
